db-commons/src/main/resources/reference.conf

```diff
@@ -119,6 +119,7 @@ bitcoin-s {
     # this config key is read by Slick
     db {
       name = oracle
+      path = ${bitcoin-s.datadir}/oracle/
```

Comment:
Copy to example config on site, let's try and keep the example config on the site consistent with our default config in the code base